zebra-rpc/src/config/mining.rs

```diff
@@ -46,9 +46,6 @@ impl Config {
     /// This is used to ignore the mining section of the configuration if the feature is not
     /// enabled, allowing us to log a warning when the config found is different from the default.
     pub fn skip_getblocktemplate(&self) -> bool {
-        #[cfg(feature = "getblocktemplate-rpcs")]
-        return false;
-        #[cfg(not(feature = "getblocktemplate-rpcs"))]
-        return true;
+        !cfg!(feature = "getblocktemplate-rpcs")
```

Comment:
Optional/Nitpick/Subjective:

```suggestion
        cfg!(not(feature = "getblocktemplate-rpcs"))
```

Reply:
I think it's clear either way